### PR TITLE
bicleaner-ai parallelism fixes

### PR DESCRIPTION
**Pull request by XapaJIaMnu**
How does this look to you @eu9ene 
I kept the CUDA_VISIBLE_DEVICES bit unchanged, but even if it is not exported, we should still be able to get the variable from inside the script. I want to test it before merging though.

EDIT:
Seems both cases are working now.

### CODE REVIEW

File: pipeline/bicleaner/bicleaner.sh [hunk 1]
```diff
@@ -44,10 +44,32 @@ else
...
+       #Convert CUDA_VISIBLE_DEVICES to an array
+       export CUDA_VISIBLE_ARRAY=($CUDA_VISIBLE_DEVICES)
+       #Turn on tensorflow logging in bicleaner-ai
+       export TF_CPP_MIN_LOG_LEVEL=0
+       #This function expects a bicleaner yaml and a 1-based index into the CUDA_VISIBLE_ARRAY
+       #Example: /mnt/nanna0/nbogoych/data/data/fr-en/fr-en-prod/biclean/pack/metadata.yaml index_in_CUDA_VISIBLE_ARRAY+1
+       biclean() {
+               export CUDA_VISIBLE_ARRAY=($CUDA_VISIBLE_DEVICES)
+               export CUDA_VISIBLE_DEVICES=${CUDA_VISIBLE_ARRAY[$(($2-1))]}
+               bicleaner-ai-classify --scol 2 --tcol 1 - - $1
```
Comment (eu9ene) [inline pipeline/bicleaner/bicleaner.sh:61]: should be `--scol ${scol} --tcol ${tcol}`

**Comment by eu9ene** [comment]
`CUDA_VISIBLE_DEVICES={gpus}` will have to go away anyway, otherwise, it will override what slurm sets. I can fix it separately.

**Comment by XapaJIaMnu** [comment]
I think it is reasonable to expect that a slurm user won't specify any other parameter other than num_gpus?

**Comment by eu9ene** [comment]
> I think it is reasonable to expect that a slurm user won't specify any other parameter other than num_gpus?

yes but the actual slurm request depends on `resources: gpus:N`. Maybe we don't have use cases to use less than `num_gpu` now, but it might be useful for evaluation for example. It's fast and it's probably easier to allocate 1 GPU on a cluster than say 8.

**Review by eu9ene — CHANGES_REQUESTED** [review]

File: pipeline/bicleaner/bicleaner.sh (hunk 1 above)
Comment (XapaJIaMnu) [reply pipeline/bicleaner/bicleaner.sh]: Fixed.

**Review by eu9ene — APPROVED** [review]

**Comment by eu9ene** [comment]
ok, merging then. If it works for you, I can fix GPU settings in a separate PR.

**Pull request merged**